lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -139,17 +139,24 @@ class ElementScreenshotRenderer {
    * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
    */
   static installOverlayFeature(dom, templateContext, fullPageScreenshot) {
-    const reportEl = dom.find('.lh-report', dom.document());
-    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
-    if (reportEl.classList.contains(screenshotOverlayClass)) return;
-    reportEl.classList.add(screenshotOverlayClass);
+    const topbarEl = dom.find('.lh-topbar', dom.document());
+    const containerEl = topbarEl.parentElement;
+    if (!containerEl) throw new Error('could not find parent element');
 
-    const maxLightboxSize = {
-      width: dom.document().documentElement.clientWidth,
-      height: dom.document().documentElement.clientHeight * 0.75,
-    };
+    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
+    if (containerEl.classList.contains(screenshotOverlayClass)) return;
+    containerEl.classList.add(screenshotOverlayClass);
 
     dom.document().addEventListener('click', e => {
+      const maxLightboxSize = {
+        width: dom.document().documentElement.clientWidth,
+        height: dom.document().documentElement.clientHeight * 0.75,
+      };
+      if (dom.isDevTools()) {
+        maxLightboxSize.width = containerEl.clientWidth;
```

Comment:
is there a reason we don't always do the `Math.min` of these? is the below correct?

In devtools:

- the container is the inner area available to our pane
- the document is the entire devtools area

In standalone:
- the container is the very tall lh-report element
- the document is the window

Reply:
> is the below correct?

yes

> is there a reason we don't always do the Math.min of these?

That would work. I find this easier to reason about though. Although using the min would possible benefit other embedders..is that your goal here?